app/controllers/application_controller.rb

```diff
@@ -29,20 +30,6 @@ class BadRequest < RuntimeError; end
   end
 
   def paginate(scope)
-    raise BadRequest, 'Both page and offset pagination is not allowed' if page_pagination? && offset_pagination?
-
-    limit_query(scope.paginate(:per_page => params[:per_page] || 10, :page => params[:page] || 1))
```

Comment:
I'd keep the behaviour similar to the original pagination and provide default values for limit and offset